src/CommunityToolkit/Xamarin.CommunityToolkit.UnitTests/EnableInitOnlyProperties.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace System.Runtime.CompilerServices
+{
+    public record IsExternalInit;
```

Comment:
Is this to enable `init` property setters?

Reply:
Yup! https://stackoverflow.com/a/66136438/5953643

I had intended on using records, but ended up removing them.

We can remove this if we want, although it doesn't harm anything and will enable records for Unit Tests in the future.